README.md

```diff
@@ -159,9 +159,9 @@ To run tests, run the following command:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215242413-73e794ac-51a8-47e2-8f23-42f6e8522de8.png)
- [ ] Your issues page link is broken :fearful: Kindly fix it to make your readme descriptive :+1: